scripts/orig_scripts/txt2img.py

```diff
@@ -240,11 +249,17 @@ def forward(self, x, sigma, uncond, cond, cond_scale):
 
     start_code = None
     if opt.fixed_code:
-        start_code = torch.randn([opt.n_samples, opt.C, opt.H // opt.f, opt.W // opt.f], device=device)
+        shape = [opt.C, opt.H // opt.f, opt.W // opt.f]
+        if device.type == 'mps':
+            start_code = torch.randn(rand_size, device='cpu').to(device)
+        else:
+            torch.randn(rand_size, device=device)
```

Comment:
this does fix the issue:

```suggestion
        shape = [opt.n_samples, opt.C, opt.H // opt.f, opt.W // opt.f]
        if device.type == 'mps':
            start_code = torch.randn(shape, device='cpu').to(device)
        else:
            torch.randn(shape, device=device)
```